src/gulcalc/gulcalc.cpp

```diff
@@ -798,6 +885,8 @@ void gulcalc::mode1()
 	char* rec = new char[max_recsize];
 	damagecdfrec* d = (damagecdfrec*)rec;
 
+	if (benchmark_) OutputBenchmark(0, true);
+
 	for (;;)
```

Comment:
What makes it exit this loop?

Reply:
When the input stream ends. See L893-894:
```
                bool bSuccess = iGetrec_(p, sizeof(damagecdfrec));
                if (bSuccess == false) break;
```

Reply:
That's outside of the for-loop though. It ends at 824

Reply:
My bad, the diff view had hidden some code I missed